### DISCUSSION TITLE
netlink for acpi

### DISCUSSION THREAD

**Discussion by rupansh** [rust-netlink/.github#7] [Show and tell]
Hello,
I have implemented parsing of netlink messages for the ACPI netlink interface

I am currently using it with my battery notification app: https://github.com/rupansh/batmon/blob/master/src/batstream/acpi/acpi_event.rs

**Comment by cathay4t** [comment]
Good to know ACPI is using netlink also.

If you want the parsing code been placed into `rust-netlink` organization. Please create `netlink-packet-acpi` pull request to https://github.com/rust-netlink/new-crate-review
